docs/DeveloperGuide.adoc

```diff
@@ -279,13 +279,15 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* is a computer science student in NUS
+* is a visual learner that benefits from using flashcards
+* can benefit from better organization of notes
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: integrate flashcards and note compilers for CS students better than a typical mouse/GUI driven app
 
 [appendix]
 == User Stories
```

Comment:
- Format is correct and all three parts are present
- Benefits match the functions
- Good variety of low, medium and high priority stories